uuu/archives/agm/2001/agenda/index.md

```diff
@@ -1,32 +0,0 @@
----
```

Comment:
Don't delete stuff under `uuu/archives`, moving it is fine but deleting all this content is not.

Reply:
The content is still archived in git and as the layout matures we will find a sensible place to put it. We've achieved the goals of the first sprint, if you will, which was to get a better-looking, more modern website made. I do not see an urgent need to keep those minutes up, but would anticipate restoring them soon, following future changes. Maybe set this as an issue in the issue tracker for future attention?

Reply:
Okay please add a new issue to the issue tracker stating that this needs to be fixed.